pybamm/models/submodels/interface/open_circuit_potential/__init__.py

```diff
@@ -2,3 +2,4 @@
 from .single_ocp import SingleOpenCircuitPotential
 from .current_sigmoid_ocp import CurrentSigmoidOpenCircuitPotential
 from .msmr_ocp import MSMROpenCircuitPotential
+from .plett_ocp import PlettOpenCircuitPotential
```

Comment:
similarly, I think we should try to give the class a name that at least hints at the physics rather than the author

Reply:
```suggestion
from .dchs_ocp import DCHSOpenCircuitPotential
```